code/modules/mining/laborcamp/laborstacker.dm

```diff
@@ -20,6 +22,16 @@
 	Radio.listening = FALSE
 	locate_stacking_machine()
 
+	if(!GLOB.labor_sheet_values.len)
+		var/sheet_list = list()
+		for(var/sheet_type in subtypesof(/obj/item/stack/sheet))
+			var/obj/item/stack/sheet/sheet = sheet_type
+			if(!initial(sheet.point_value))
+				continue
+			sheet_list[initial(sheet.name)] = initial(sheet.point_value)
+		for(var/sheet_name in sortList(sheet_list))

```

Comment:
Sorts by name rather than point value like it used to be. If anyone really cares I'll try to find some way to fix it.